tests/test_skip_repeats_queue.py

```diff
@@ -81,3 +88,14 @@ def test_consecutives_allowed_across_empties():
     q.put(e1)  # this repeat is allowed because 'last' added is now gone from queue
     assert e1 == q.get()
     assert q.empty()
+
+
+@cpython_only
+def test_eventlet_monkey_patching():
```

Comment:
TODO for me: test all functions using the raw and money patched objects with fixtures. Just this test is not enough.

Reply:
Looks good for me, so now I will check how tests run on Windows and patch it if need.